<launch> 
  <node name="camera_node" pkg="usb_cam" type="usb_cam_node">
    <param name="camera_info_url" value="package://usb_cam/launch/ov2610.yaml"/>
    <param name="video_device" value="/dev/video2"/>
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" /> 
    <param name="framerate" value="15" />
    <param name="time_delay" value="50" />
    <param name="camera_frame_id" value="ov2610" />
    <param name="io_method" value="mmap"/>
  </node>
</launch>
